code/modules/unit_tests/metabolizing.dm

```diff
@@ -11,11 +11,9 @@
 	user.Life()
 
 	TEST_ASSERT(user.reagents.has_reagent(meth), "User does not have meth in their system after consuming it")
-	TEST_ASSERT(user.has_movespeed_modifier(/datum/reagent/drug/methamphetamine), "User consumed meth, but did not gain movespeed modifier")
 
 	user.Life()
 	TEST_ASSERT(!user.reagents.has_reagent(meth), "User still has meth in their system when it should've finished metabolizing")
-	TEST_ASSERT(!user.has_movespeed_modifier(/datum/reagent/drug/methamphetamine), "User still has movespeed modifier despite not containing any more meth")
 
```

Comment:
👍

Let's see what I can find for it.

Reply:
I have implemented a new "fun" type chem called Badminordrazine to use for the check. It also doubles as an adminbus chem to smite players with that isn't so over-the-top it can't see use. 

*It can also show up in floorpills*

Reply:
just change it to crank and check if they have TRAIT_NOBLOCK. no need to create new reagent just for that